az_quickstart.install

```diff
@@ -79,10 +79,7 @@ function az_quickstart_requirements($phase) {
 
     // Gather all deprecated modules that are enabled.
     $azqs_deprecated_modules = [
-      'background_image_formatter',
       'block_content_permissions',
-      'ckeditor',
-      'entity_browser',
       'media_library_theme_reset',
```

Comment:
FWIW, we've actually added 2 different DB updates that uninstall this module (since it was unsupported by the maintainer) so I think it should be safe to remove at this point.